fuse/dl/lightning/pl_module.py

```diff
@@ -82,8 +82,12 @@ def __init__(
         """
         super().__init__(**kwargs)
 
+        if (save_arguments or save_model) and (model_dir is None):
```

Comment:
Are you avoiding from lightning logging? or maybe it's configurable in trainer?

Reply:
Yes, users can either only use lightning logging, or not log at all (which can be useful for debugging).